_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,161 @@
+/* ============================
+-------------------------------
+ 1. Grid
+ 2. Fonts
+ 3. Colors (Palette)
+ 4. Colors (Application)
+ 5. Timing
+-------------------------------
+============================ */
+
+$baseline:                      20px;
```

Comment:
NIT: alignment of values is nice, but this is a ton of whitespace between all names/values. Please tighten.